hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -170,10 +169,10 @@ public RpcClient(ConfigurationSource conf, String omServiceId)
         OzoneManagerProtocol.class, conf
     );
     dtService = omTransport.getDelegationTokenService();
-    ServiceInfoEx serviceInfoEx = ozoneManagerClient.getServiceInfo();
+    String certificate = ozoneManagerClient.getCaCertificate();
     String caCertPem = null;
     if (OzoneSecurityUtil.isSecurityEnabled(conf)) {
-      caCertPem = serviceInfoEx.getCaCertificate();
+      caCertPem = certificate;
```

Comment:
how about 
```
caCertPem = ozoneManagerClient.getCaCertificate();
```

only invoke when security is enabled.

Reply:
CI does not passed if we do this, maybe CI want ozoneManagerClient.getCaCertificate throw exception even if security is not enabled. I will find the reason.